pages/_app.js

```diff
@@ -20,6 +20,14 @@ export default function App({ Component, pageProps }) {
       );
     });
   }
+  function handleChangeCard(newCard) {
```

Comment:
As a name for `newCard`, `editedCard` or `updatedCard` would probably be more appropriate.